apps/coordinator/src/components/Wallet/TransactionsTab/FeeBumping/components/hooks.ts

```diff
@@ -98,15 +105,32 @@ export const useAnalyzeTransaction = (
         requiredSigners,
         totalSigners,
         addressType: addressType as MultisigAddressType,
+        ...(changeOutputIndex !== undefined && { changeOutputIndex }),
       });
-      return analyzer.analyze();
+      let cpfpData = null;
+      try {
+        // added here so that in case `analyzer.cpfpFeeRate` throws error it gets catched
```

Comment:
```suggestion
        // added here in case `analyzer.cpfpFeeRate` throws an error
```

Reply:
why not let the error get thrown though and get caught and set on line 127 so it can be surfaced?